api/src/main/java/org/pentaho/platform/api/engine/IAuthorizationActionService.java

```diff
@@ -0,0 +1,121 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+package org.pentaho.platform.api.engine;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import org.apache.commons.lang.StringUtils;
```

Comment:
I would rather not add a dependency on commons lang just to prevent an explicit not null or empty check on a string.

Reply:
This is already used in many places in this module.